go-controller/pkg/node/gateway_init.go

```diff
@@ -177,7 +177,21 @@ func getGatewayNextHops() ([]net.IP, string, error) {
 				}
 			}
 		}
+	} else if config.Gateway.Mode == config.GatewayModeLocal {
```

Comment:
I wonder why would we honor a next hop configuration in local gateway mode.

Reply:
Yeah, I think we can ignore the user-configured next-hop in local gateway mode.